internal/policy/src/main/resources/policy/checkstyle.xml

```diff
@@ -64,8 +70,13 @@
             <property name="sortStaticImportsAlphabetically" value="true"/>
         </module>
         <!--<module name="InnerAssignment"/>-->
+        <module name="Indentation"/>
         <module name="InnerTypeLast"/>
         <module name="InterfaceIsType"/>
+        <module name="JavadocStyle">
+            <property name="checkFirstSentence" value="false"/>
```

Comment:
are those changing the default setting or just showing the default?

Reply:
The default is `checkFirstSentence = true` which forces you to end the first Javadoc line with a sequence satisfying `"([.?!][ \t\n\r\f<])|([.?!]$)"`.
Let's say you break a lengthy sentence onto the next line, you'd get a Checkstyle violation.

```
        String commentText = "Henning wrote this";
        Pattern endOfSentenceFormat = Pattern.compile("([.?!][ \t\n\r\f<])|([.?!]$)");
        if (!commentText.isEmpty()
            && !endOfSentenceFormat.matcher(commentText).find()
            && !commentText.startsWith("{@inheritDoc}")) {

            System.err.println("Fail: " + commentText);
        }
```

Reply:
I think Checkstyle themselves may have changed the default from `false` to `true`, but I did not dig into it any deeper.